spec/02-integration/04-admin_api/08-targets_routes_spec.lua

```diff
@@ -753,7 +753,7 @@ describe("Admin API #" .. strategy, function()
           },
           headers = { ["Content-Type"] = "application/json" }
         })
-        assert.response(res).has.status(204)
+        assert.response(res).has.status(200)
```

Comment:
In addition to the status code, should we check for the JSON response as well to make sure it contains a valid entity?